tests/tests_combination.yml

```diff
@@ -52,7 +52,6 @@
           - name: "{{ __test_tag }}"
             type: files
             input_log_path: "{{ __test_inputfiles_dir }}/*.log"
-            endmsg_regex: xyz
```

Comment:
RHEL 7 does not support `endmsg_regex`

Reply:
> RHEL 7 does not support `endmsg_regex`

ok - then we should handle this here by omitting `endmsg_regex` only on RHEL7, or in the template

Reply:
I'm omitting this in the test. If users set `endmsg_regex` they should get a message in /var/log/messages that this parameter is not known.